### DISCUSSION TITLE
VDH 8.2.0.23 / CoApp 2.0.17 / Firefox / Windows 11 – Can't select destination directory

### DISCUSSION THREAD

**Discussion by HereButNotaGit** [aclap-dev/video-downloadhelper#169] [Help and Q&A] [answered]
### Your question

I'm trying to download a video from a site where I've previously had no problems.  VDH presents the CoApp screen to select the destination directory.  That's fine.  As usual, I want to download into a multilevel directory structure on a dedicated drive (T:).  I can drill down into a directory (SOURCES) in the root directory of T:.  However, if I then try to drill down into a subdirectory under SOURCES (something I do every time), I get bounced straight back into SOURCES as the current working directory.  I can't drill down to the desired destination directory.  To resolve the problem, I uninstalled 8.2.0.23, and installed 8.0.0.0.  Then everything worked fine as expected.

This is why I archive full offline installers for anything that I can, because you never know when you might need to backlevel a software item, when an "upgrade" to a software item leaves it broken.  I've suffered breakage issues with some of the 8.*.*.* "upgrades", but 8.0.0.0 is so far working reliably for me.

### Addon & CoApp version and details

VDH 8.2.0.23 / CoApp 2.0.17 / Firefox 122.0.1 (latest) / Windows 11

**Comment by paulrouget** [comment, marked as answer]
This is fixed in the upcoming release.
